lib/dry/system/plugins/bootsnap.rb

```diff
@@ -5,11 +5,9 @@ module System
     module Plugins
       module Bootsnap
         DEFAULT_OPTIONS = {
-          load_path_cache: true,
-          disable_trace: true,
+          load_path_cache: false,
```

Comment:
had to change this from true to false to get CI passing once again. with the option as true, some specs would fail with the error below that I haven't been able to figure out what is causing it
```
  11) Container / Imports / Exports exports configured as a list of keys importing container is lazy loading does not finalize either container
      Failure/Error: raise ComponentNotLoadableError.new(component, e)
      
      Dry::System::ComponentNotLoadableError:
        Component 'exportable_component_a' is not loadable.
        Looking for Test::ExportableComponentA.
      
        Did you mean?  ExternalComponents
      
                    raise ComponentNotLoadableError.new(component, e)
                    ^^^^^
      # ./lib/dry/system/loader.rb:71:in `rescue in constant'
      # ./lib/dry/system/loader.rb:64:in `constant'
      # ./lib/dry/system/loader.rb:49:in `call'
      # ./lib/dry/system/component.rb:64:in `instance'
      # ./lib/dry/system/container.rb:639:in `block in load_local_component'
      # ./lib/dry/system/container.rb:493:in `resolve'
      # ./lib/dry/system/container.rb:516:in `key?'
      # ./lib/dry/system/importer.rb:109:in `block in build_merge_container'
      # ./lib/dry/system/importer.rb:108:in `each'
      # ./lib/dry/system/importer.rb:108:in `each_with_object'
      # ./lib/dry/system/importer.rb:108:in `build_merge_container'
      # ./lib/dry/system/importer.rb:86:in `import_keys'
      # ./lib/dry/system/importer.rb:69:in `import'
      # ./lib/dry/system/container.rb:648:in `load_imported_component'
      # ./lib/dry/system/container.rb:626:in `load_component'
      # ./lib/dry/system/container.rb:491:in `resolve'
      # ./spec/integration/container/importing/exports_spec.rb:76:in `block (4 levels) in <top (required)>'
      # ------------------
      # --- Caused by: ---
      # NameError:
      #   uninitialized constant Test::ExportableComponentA
      #   
      #         Object.const_get(input, false)
      #               ^^^^^^^^^^
      #   Did you mean?  ExternalComponents
      #   ./lib/dry/system/loader.rb:67:in `constant'


```